example/pubspec.yaml

```diff
@@ -25,6 +25,10 @@ dependencies:
 dependency_overrides:
   flutter_map:
     path: ../
+  flutter_map_geojson:
+    git:
+      url: https://github.com/JaffaKetchup/flutter_map_geojson.git
+      ref: fm-v8
```

Comment:
Maybe it's worth to think about a first party solution if we rely that havily on the package?

Reply:
It would be nice, but I don't have time to do that right now. I guess the ideal thing to do would be to preprocess the geojson data into some form that's  very easy to decode (given we don't use very much of its functionality, maybe just a list of numbers or something).
If the PR I make isn't accepted I can look into this.

Reply:
This comment is no merge blocker. Just a note 😄

Reply:
> preprocess the geojson data into some form that's very easy to decode

Yes that would be ideal.